gammapy/modeling/models/cube.py

```diff
@@ -252,7 +252,7 @@ def contributes(self, mask, margin="0 deg"):
 
         if mask.geom.is_region and mask.geom.region is not None:
             geom = mask.geom.to_wcs_geom()
-            mask = geom.region_mask([mask.geom.region])
+            mask = ~geom.region_mask([mask.geom.region])
```

Comment:
There is a related test-fail: https://github.com/gammapy/gammapy/pull/3484/checks?check_run_id=3414904505#step:9:1975

Maybe it's good to add a unit test for this as well...